### DISCUSSION TITLE
Is these features possible on Powergrid ?

### DISCUSSION THREAD

**Discussion by thanjeys** [Power-Components/livewire-powergrid#661] [Ideas]
Hi, 

Its a good component. Thank you
but still i am facing problem to use this. 
Can author or any tell me 
Is these features are available on powergrid and Solution pls ? 

I am facing these are problems on powergrid. SO please give me solutions for this?

1) Need Remove (Contains, is start with, not start) Dropdown on Search Input ? 

2) I Need to add own search option( Instead of date picker,) I want to give option like ( today, this mont, thisweek, this year). i.e Overriding Global Search.

3) Dependent dropdown option 

4) Export PDF format 

5) Show records from navigating dashboard page ex :  Showing Today sales : 20000, If user click this, our datatable will show only today records 

I Have been waiting for solutions. 

Thank you very much !!

**Comment by luanfreitasdev** [comment]
Hello.

Thanks for using powergrid, it's a growing package and your feedback is very helpful.

He follows:
1 - Scheduled to be released in version 4.
2 - It has no scope.
3 - Scheduled to be released in version 4.
4 - It is currently not possible, but in version 4 a new exporter can be included as it will not depend on packages in composer. (I'm grateful for a Pr - it depends https://github.com/Power-Components/livewire-powergrid/pull/621).
5 - I didn't quite understand.

[1, 3 and 4]. Has breaking changes
[2 and 5] - Please specify more in detail so we can understand the situation. You can also sponsor development or submit a PR.

Thank you very much

**Reply by thanjeys** [reply]
Hello,

Thanks for your response. 

Custom Filter Option is required: 

For example, If User wants to see Last year Records means they should select Date from to end. Its not user friendly 
So i can give option Last year, Last 6 month as  a Dropdown. If user select option, Records would have shown 


If you still didnt get it, let me know

Thanks
